File: OBI/2021/M2/level_3/square_and_cube.py
import math

def main()->None:
    x=int(input())
    y=int(input())

    x_range=math.ceil(x**(1/6))
    y_range=math.floor(y**(1/6))


    count=y_range-x_range+1

    print(count)

if __name__=="__main__":
    main()

File: OBI/2021/M2/level_3/square_and_cube.cpp
#include<iostream>
#include<cmath>
//
int main(){
    int n,m;
    std::scanf("%i %i",&n,&m);

    int n_range=std::ceil(std::pow(n,1.0/6.0));
    int m_range=std::floor(std::pow(m,1.0/6.0));

    int count=m_range-n_range+1;

    printf("%i\n",count);

    return 0;
}
